### PR TITLE
TypeScriptize TraceJsonUploader and TraceIdSearch

### PR DESCRIPTION
**Pull request by tacigar**

### CODE REVIEW

File: zipkin-lens/src/translations/en/messages.json [hunk 1]
```diff
@@ -42,7 +42,6 @@
   "Zipkin Home": "",
   "hide annotations": "",
   "show all annotations": "",
-  "trace id...": "",
```
Comment (codefromthecrypt) [inline zipkin-lens/src/translations/en/messages.json:45]: I assume this is orphaned

**Review by codefromthecrypt — COMMENTED** [review]
Can you mention motivation for this change? It seems like reduction of code is the intent.

Also, is there any visual difference as a result?

**Review by anuraaga — APPROVED** [review]
Probably mostly for consistent tooling / code. Didn't see any differences when reviewing seems nice.

**Comment by codefromthecrypt** [comment]
Thanks!

**Pull request merged**